server/docker-entrypoint.sh

```diff
@@ -4,5 +4,4 @@ if [ $KEYCLOAK_USER ] && [ $KEYCLOAK_PASSWORD ]; then
     keycloak/bin/add-user-keycloak.sh --user $KEYCLOAK_USER --password $KEYCLOAK_PASSWORD
 fi
 
-exec /opt/jboss/keycloak/bin/standalone.sh $@
-exit $?
+exec /opt/jboss/keycloak/bin/standalone.sh $@
```

Comment:
`exec` system call replaces the current shell with the specified program so exit will never actually be called.